recipes/opensubdiv/all/conanfile.py

```diff
@@ -0,0 +1,165 @@
+    name = "opensubdiv"
+    license = "LicenseRef-LICENSE.txt"
+    homepage = "https://github.com/PixarAnimationStudios/OpenSubdiv"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "An Open-Source subdivision surface library"
+    topics = ("cgi", "vfx", "animation", "subdivision surface")
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "with_tbb": [True, False],
+        "with_opengl": [True, False],
+        "with_omp": [True, False],
+        "with_cuda": [True, False],
+        "with_clew": [True, False],
+        "with_opencl": [True, False],
+        "with_dx": [True, False],
+        "with_metal": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "with_tbb": False,
+        "with_opengl": False,
+        "with_omp": False,
+        "with_cuda": False,
+        "with_clew": False,
+        "with_opencl": False,
+        "with_dx": False,
+        "with_metal": False,
+    }
+
+    short_paths = True
+    generators = "cmake"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    @property
+    def _minimum_cpp_standard(self):
+        if self.options.get_safe("with_metal"):
+            return 14
+        return 11
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "15",
+            "gcc": "5",
+            "clang": "11",
+            "apple-clang": "11.0",
+        }
+
+    def export_sources(self):
+        self.copy("CMakeLists.txt")
+
+    def config_options(self):
+        if self.settings.os != "Windows":
+            del self.options.with_dx
+        if self.settings.os != "Macos":
+            del self.options.with_metal
+
+    def configure(self):
+        if self.settings.os == "Windows":
+            del self.options.shared
+
+    def requirements(self):
+        if self.options.with_tbb:
+            self.requires("onetbb/2020.3")
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, self._minimum_cpp_standard)
+        min_version = self._minimum_compilers_version.get(str(self.settings.compiler))
+        if not min_version:
+            self.output.warn("{} recipe lacks information about the {} compiler support.".format(self.name, self.settings.compiler))
+        else:
+            if tools.Version(self.settings.compiler.version) < min_version:
+                raise ConanInvalidConfiguration(
+                    "{} requires C++{} support. The current compiler {} {} does not support it.".format(self.name, self._minimum_cpp_standard, self.settings.compiler, self.settings.compiler.version)
+                )
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
```

Comment:
```suggestion
        get(self, **self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
```
Conan v2 Linter wants new tools